main.go

```diff
@@ -117,6 +118,12 @@ func main() {
 			leaderElectionID = operatorName
 		}
 	}
+	
```

Comment:
Add const here and address impl t sdk/internal lib it will be valid for Helm/Ansible.